contrib/signing/DEBIAN/control.in

```diff
@@ -0,0 +1,8 @@
+Package: install-signed-packages
+Version: @VERSION@
+Section: development
+Priority: optional
+Architecture: any
+Maintainer: OrbitProfiler Team <orbitprofiler-team@google.com>
```

Comment:
Isn't this orbitprofiler-eng@google.com?

Reply:
Yes. you're right. Just blindly followed the examples. Thanks, will fix it.